src/index.ts

```diff
@@ -1,16 +1,16 @@
-type EventType = string | symbol;
+export type EventType = string | symbol;
```

Comment:
@developit I guess now we have a `Map` under the hood this could technically be anything; so it might be worth removing this type and allowing `any` as the type? WDYT?

Reply:
I think the TS typings should stay aligned with Node's [EventEmitter.emit](https://nodejs.org/api/events.html#events_emitter_emit_eventname_args), which states `string | symbol`.